components/cam/src/physics/clubb/advance_xm_wpxp_module.F90

```diff
@@ -4327,9 +4330,9 @@ pure function damp_coefficient( coefficient, Cx_Skw_fnc, max_coeff_value, &
       Cx_Skw_fnc            ! Initial skewness function before damping
 
     ! Local variables
-    real( kind = core_rknd ), parameter :: &
+!    real( kind = core_rknd ), parameter :: &
```

Comment:
Same here. We can probably remove all commented out lines including comments, since they are not relevant anymore.

Reply:
Thank you @singhbalwinder for the suggestion. I removed the unnecessary lines and comments. I will clean up the unnecessary lines in the future commits as well.